app/src/main/java/com/example/vande/scouting2017/TeleopActivity.java

```diff
@@ -226,8 +287,7 @@ public void saveData(View view) throws IOException {
             Toast.makeText(getApplicationContext(),"message Saved", Toast.LENGTH_LONG).show();
 
 
-
-                Intent intent = new Intent(context, AutonActivity.class);
+                Intent intent = new Intent(this, AutonActivity.class);
```

Comment:
Remove the intent and startActivity and below the clearData call put finish();

Starting activity again is going to keep creating intents and a backlog of activites so your back button will not function right.

Reply:
This needs removed